spec/models/game_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
```

Comment:
Can you test the `full_name` method as well? Make sure you create Users with weird first and last names. Try to break it.